R/create_map.R

```diff
@@ -16,7 +23,6 @@ create_map <- function(png_filename, show_bridge_openings = TRUE) {
     ggmap::get_map(bbox, maptype = "toner_stamen", quiet = TRUE))
   p <- ggmap::ggmap(groningen)
   if (show_bridge_openings) {
-    data <- informalr::get_bridge_openings()
```

Comment:
Deleting this line seems like an error to me. I suggest to put it back first :grin: :+1: